src/components/Carousel/Carousel.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect } from "react";
+import "./Carousel.scss";
+
+const Carousel = () => {
+  const [curIndex, setCurIndex] = useState(1);
+  const [action, setAction] = useState(1);
+  const [isTransition, setIsTransition] = useState(false);
+
+  useEffect(() => {
+    const timer = setInterval(() => setCurIndex(curIndex + 1), 4000);
+    return () => {
+      clearInterval(timer);
+    };
+  });
+
+  useEffect(() => {
+    setTimeout(() => {
+      console.log("hi");
+      setIsTransition(false);
+    }, 1000);
+    // setTimeout(() => {
+    //   setAction(0);
+    // }, 500);
+    curIndex === 0 &&
```

Comment:
```suggestion
const startIndex = 0
    curIndex === startIndex &&
```
* 0도 좋지만, 조건으로 좀더 쉽게 이해할 수 있는 변수로 선언되는게 좋겠습니다

Reply:
감사합니다!